services/src/main/java/org/keycloak/services/ServicesLogger.java

```diff
@@ -253,7 +253,7 @@ public interface ServicesLogger extends BasicLogger {
     @Message(id=52, value="Failed processing type")
     void failedProcessingType(@Cause Exception e);
 
-    @LogMessage(level = WARN)
+    @LogMessage(level = DEBUG)
```

Comment:
I agree to change this to DEBUG, and also to eventually remove it in a future release as the `LOGIN_ERROR` event which is also logged contains the same information. 

IMHO you could go as far as removing it. Either way, it needs to be mentioned in the release notes, as people might depend on this log message, and should recommend to use the `LOGIN_ERROR` event instead.

Reply:
I've removed the log message completely. 

Now I must confess, I'm not aware how I can add a note to the release notes?

Reply:
I'll prepare a suggestion and push it to this branch. Then you can review it.